applications/nrf_desktop/src/modules/ble_bond.c

```diff
@@ -81,6 +81,9 @@ static u8_t cur_peer_id;
 static u8_t tmp_peer_id;
 static bool dongle_peer_selected_on_init;
 
+static bool storage_cur_peer_id_invalid;
```

Comment:
Please keep these close to related variables (maybe within single structure). I would prefer reversed logic (by default invalid - after settings are loaded reset is done if no valid settings were found).
e.g. 
```
static u8_t cur_peer_id;
static bool cur_peer_id_valid;
```
or
```
static struct {
   u8 data;
   bool valid;
} cur_peer_id;
```

Reply:
Ok, moved.
Overwrite is done only if invalid settings were found (its not necessary if there is no data in storage).
It's why I would prefer to use `invalid` flag, which is set to `false` by default.

Reply:
Discussed offline